src/lib/apis/atoma/zklogin.ts

```diff
@@ -0,0 +1,93 @@
+import { SuiClient } from '@mysten/sui/client';
+import { Ed25519Keypair } from '@mysten/sui/keypairs/ed25519';
+import {
+	genAddressSeed,
+	generateNonce,
+	generateRandomness,
+	getExtendedEphemeralPublicKey,
+	getZkLoginSignature,
+	jwtToAddress
+} from '@mysten/zklogin';
+
+const LOCAL_STORAGE_SALT = 'ZKLOGIN-SALT';
+// const LOCAL_STORAGE_NONCE = 'ZKLOGIN'-NONCE';
+const LOCAL_STORAGE_SECRET_KEY = 'ZKLOGIN-SECRET_KEY';
+const LOCAL_STORAGE_RANDOMNESS = 'ZKLOGIN-RANDOMNESS';
+const LOCAL_STORAGE_MAX_EPOCH = 'ZKLOGIN-MAX_EPOCH';
+const LOCAL_STORAGE_ZK_ADDRESS = 'ZKLOGIN-ZK_ADDRESS';
+// const LOCAL_STORAGE_ID_TOKEN = 'ZKLOGIN'-ID_TOKEN';
+const LOCAL_STORAGE_ZKP = 'ZKLOGIN-ZKP';
+
+const suiClient = new SuiClient({
+	url: 'https://api.shinami.com:443/node/v1/sui_testnet_e82cc48bea6170db8d058710bb4c10fa'
```

Comment:
Please add an environment variable here (don't hardcode rpc nodes, especially for testnet).

Reply:
added both, the prover url was also hardcoded